### PR TITLE
Snapshot trace docs

### PR DESCRIPTION
**Pull request by winkerVSbecks**
* Refactored the ["Troubleshooting Snapshots" page](https://deploy-preview-572--chromatic-docs.netlify.app/docs/troubleshooting-snapshots)
* Added a [section on snapshot trace](https://deploy-preview-572--chromatic-docs.netlify.app/docs/troubleshooting-snapshots/#how-to-use-the-snapshot-trace)

@jwir3 @alright-fine let me know if you all have any suggestions on what to add/remove

### CODE REVIEW

File: src/content/snapshot/troubleshooting-snapshots.md [hunk 1]
```diff
@@ -7,67 +7,61 @@ sidebar: { order: 3, label: "Troubleshooting" }
...
+The Snapshot Trace Viewer lets you explore recorded traces of tests rendered and snapshotted in the Chromatic Capture Cloud. It captures network requests, console logs, and other debugging information, helping you identify the root cause of rendering issues.
 
-- **Intentional randomness**: Some stories may render unpredictably intentionally. If this is the case you may want to [ignore the story](/docs/disable-snapshots#with-storybook) from UI Tests and move on.
-  If you still need inconsistent elements for local development purposes inside Storybook, you can use `isChromatic()` exported from [our package](/docs/ischromatic) to apply the solutions above only when in the Chromatic environment.
+Once you rerun a build, the subsequent build will feature a "Traces" column. This column links to the Trace Viewer for each snapshot in the build, with one link per enabled browser. Clicking on one of the browser buttons to open the Trace Viewer.
```
Comment (jonniebigodes) [inline src/content/snapshot/troubleshooting-snapshots.md:31]: Small typo here
```suggestion
Once you rerun a build, the subsequent build will feature a "Traces" column. This column links to the Trace Viewer for each snapshot in the build, with one link per enabled browser. Click on one of the browser buttons to open the Trace Viewer.
```

File: src/content/snapshot/troubleshooting-snapshots.md [hunk 2]
```diff
@@ -7,67 +7,61 @@ sidebar: { order: 3, label: "Troubleshooting" }
...
 
-Portals allow components to render arbitrary elements outside the parent component's initial DOM hierarchy. For example, tooltips, modals, popovers, and menus can be triggered by a nested button, but render close to the top of the DOM hierarchy using portals.
+#### Network tab analysis
 
-With Playwright and Cypress, Chromatic snapshots the entire page, including modals, dialogs, and other portaled elements. However, with Storybook, portals render outside of Storybook’s DOM tree which can lead to cut-off snapshots.
+The network tab displays the resources loaded during capture, including fonts, stylesheets, scripts, and other assets. Check for resources that failed to load or took a long time. For example, if fonts aren't incorrect or styles are missing, ensure that the font or CSS files have loaded successfully with the correct MIME type. Consider loading slow assets [statically](#serve-static-files).
 
-<details>
-<summary>Why are snapshots of portal components (tooltip, modal, popover, menu) cut off?</summary>
+For a detailed list of trace viewer features, see the [Playwright documentation](https://playwright.dev/docs/trace-viewer#trace-viewer-features).
```
Comment (jonniebigodes) [inline src/content/snapshot/troubleshooting-snapshots.md:45]: The "For a detailed list of trace viewer features...." sentence feels like it should be something that could be inside an aside, wouldn't you agree?

File: src/content/snapshot/troubleshooting-snapshots.md [hunk 3]
```diff
@@ -7,67 +7,61 @@ sidebar: { order: 3, label: "Troubleshooting" }
...
-  ],
-};
-```
+#### Screenshot metadata
+
+When Chromatic captures a screenshot, it includes metadata like viewport information and clip rectangle dimensions, providing context for the capture. Use this data to identify issues such as:
 
-## Debug snapshot rendering
+- **Responsive design issues**: Viewport information reveals the screen size used for the capture. Compare this with your breakpoints to ensure that the correct styles are applied.
+- **Element positioning problems**: The clip rectangle shows precisely what was captured. If an element is missing from the snapshot, verify if it falls within the expected clip area. For instance, Chromatic only captures DOM elements within the `#storybook-root` element for stories. Modal and dropdown components might not be captured if they fall outside the "natural" dimensions of this root element. Use [fixed-height wrappers](#use-fixed-height-wrappers-for-portal-components) to capture these portal components.
```
Comment (jonniebigodes) [inline src/content/snapshot/troubleshooting-snapshots.md:60]: Small suggestion here for a bit of clarity. Feel free to disregard
```suggestion
- **Element positioning problems**: The clip rectangle shows precisely what was captured. If an element is missing from the snapshot, verify if it falls within the expected clip area. For instance, Chromatic only captures DOM elements within the `#storybook-root` element for stories. Modal and dropdown components might not be captured if they fall outside this root element's "natural" dimensions. Use [fixed-height wrappers](#use-fixed-height-wrappers-for-portal-components) to capture these portal components.
```

**Review by jonniebigodes — APPROVED** [review]
@winkerVSbecks thanks for taking the time to put this pull request and get the trace viewer into the documentation. Appreciate it. This is shaping up nicely. I left a couple of small items for you to look into when you can

File: src/content/snapshot/troubleshooting-snapshots.md [hunk 4]
```diff
@@ -7,67 +7,61 @@ sidebar: { order: 3, label: "Troubleshooting" }
...
+Once you rerun a build, the subsequent build will feature a "Traces" column. This column links to the Trace Viewer for each snapshot in the build, with one link per enabled browser. Clicking on one of the browser buttons to open the Trace Viewer.
 
-### Serve static files
+![Build 2 was a rerun build so it has an additional "traces" column](../../images/view-trace.png)
 
-When using Playwright or Cypress, you can serve static files like fonts, images, and videos through your app server. This ensures that resources load consistently across all snapshots.
+### How to use the snapshot trace?
 
-For Storybook, use the [staticDirs](https://storybook.js.org/docs/configure/integration/images-and-assets#serving-stae-files-via-storybook-configuration) option to load static files for your stories.
+To generate these traces, Chromatic leverages Playwright's built-in [tracing capabilities](https://playwright.dev/docs/trace-viewer#network). Even if you're using Cypress or Storybook, Chromatic's uses Playwright to render and capture snapshots in its Capture Cloud. These traces capture network activity, console logs, DOM snapshots, and other debugging information.
```
[the remark this answers is not in the document]
Comment (winkerVSbecks) [reply src/content/snapshot/troubleshooting-snapshots.md]: Reworded it to use your suggestion

[the remark this answers is not in the document]
Comment (jonniebigodes) [reply src/content/snapshot/troubleshooting-snapshots.md]: Appreciate it 🙏

**Review by alright-fine — APPROVED** [review]
Nothing stands out as needing to be changed, I'm excited for this to go live!

**Pull request merged**